src/main/target/FF_FORTINIF4/config.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <platform.h>
+
+#ifdef TARGET_CONFIG
+#include "telemetry/telemetry.h"
+
+void targetConfiguration(void)
+{
+    telemetryConfigMutable()->telemetry_inverted = false;
```

Comment:
This was requested by FuriousFPV. I do taget maitenace for them. There SBUS receivers have uninverted signal output. This is to make there products more plug and play.